Dockerfile

```diff
@@ -9,28 +26,22 @@ ENV APP_BUILD_TAG ${APP_BUILD_TAG}
 ARG APP_GIT_COMMIT
 ENV APP_GIT_COMMIT ${APP_GIT_COMMIT}
 
+ENV APPUID 1000
+
 ENV RAILS_ENV production
+
 ENV PUMA_PORT 3000
 EXPOSE $PUMA_PORT
 
-WORKDIR /app
-
-# ugrade bundler to 2.1.4
-RUN gem update bundler --no-document
-
-COPY . /app
+RUN addgroup -g $APPUID -S appgroup && \
+    adduser -u $APPUID -S appuser -G appgroup -h /app
 
-RUN bundle install --verbose --without="development test" --jobs 4 --retry 3
+RUN apk add --no-cache tzdata postgresql-dev
```

Comment:
ah is it that we can either have update or no cache?

Reply:
This just prevents caching so you don't have to delete it after. Haven't covered the update yet.